db/migrate/20170605090919_add_expression_type_to_miq_alert.rb

```diff
@@ -0,0 +1,23 @@
+class ::MiqExpression; end
```

Comment:
I believe you can try serializing manually or specify a coder which will use your MiqExpression stub within the migration namespace:

https://www.viget.com/articles/how-i-used-activerecord-serialize-with-a-custom-data-type